calculator_test.go

```diff
@@ -2,23 +2,275 @@ package calculator_test
 
 import (
 	"calculator"
+	"math/rand"
 	"testing"
 )
 
-func TestAdd(t *testing.T) {
-	t.Parallel()
-	var want float64 = 4
-	got := calculator.Add(2, 2)
+func assertNumbers(want, got float64, t *testing.T) {
```

Comment:
In general we don't tend to write `AssertX...`-style functions in Go. Instead, it's much clearer to use the built-in `if` statement and the `==` operator!

Reply:
But it what my function does... are you saying I should repeat code if got != want { } on each test?

Reply:
That's right.

Reply:
it looks like code repetition for me, but I do trust you!